lab_notebook.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<p>I'd change the errors to<code>abs(1 - model_history.iloc[-1]['model_UNI_supply']/real_history.iloc[-1]['real_UNI_supply'])</code></p><p>so that we display the relative error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/uniswap/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>